Dockerfile

```diff
@@ -89,17 +135,20 @@ RUN mkdir /usr/local/share/jupyterhub/static/swan/ && \
 
 # Handlers, Spawners, Templates, ...
 RUN git clone -b $JUPYTERHUB_EXTENSIONS_TAG https://gitlab.cern.ch/swan/jupyterhub.git /srv/jupyterhub/jh_gitlab
-# Install CERN Handlers
-WORKDIR /srv/jupyterhub/jh_gitlab/CERNHandlers
-RUN pip install -r requirements.txt && \
-    python3.6 setup.py install
+# Install CERN KeyCloakAuthenticator (oauthenticator)
+WORKDIR /srv/jupyterhub/jh_gitlab/KeyCloakAuthenticator
+RUN pip3 install .
 # Install CERN Spawner
 WORKDIR /srv/jupyterhub/jh_gitlab/SwanSpawner
-RUN pip install .
+RUN pip3 install .
+# Install CERN Handlers
+WORKDIR /srv/jupyterhub/jh_gitlab/CERNHandlers
+RUN pip3 install .
+
```

Comment:
There is no install of SwanNotificationService. We don't really need it, but it's part of the system.

Reply:
since we will only have one instance of jupyterhub we don't need it in k8s/docker-compose world

Reply:
For now. The idea is to continue having notifications in the future (like nodes being removed for maintenance that might still have a pod running).

Reply:
But fine, for now we can omit it. I just didn't want the calls to a path that doesn't exist.